tests/common.sh

```diff
@@ -66,6 +66,8 @@ function build_sketches()
     local sketches=$(find $srcpath -name *.ino | sort)
     print_size_info >size.log
     export ARDUINO_IDE_PATH=$arduino
+    local pwm_phase=""
+    [ $(( $build_rem % 2 )) -eq 0 ] && local pwm_phase="--waveform_phase"
```

Comment:
Bash is a monster, isn't it?  I was about to make a comment that you wanted `==` but then realized you're generating local.

Not sure if the `local` here is needed since you already made one on the prior line.

Reply:
I removed the second local